### PR TITLE
Updated README wrt. new features for Stax.

### PR DESCRIPTION
**Pull request by dynamicwebpaige**

### CODE REVIEW

File: jax/experimental/README.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ libraries are in part about providing tools and in part about serving as
 examples for how to build such libraries using JAX. Each one is only a few
 hundred lines of code, so take a look inside and adapt them as you need!
 
+#### 👉 **Note**: These mini-libraries are meant to be an _inspiration_, but not a prescription. 
+
+To serve that purpose, it is best to keep their code samples minimal; so we generally *will not merge PRs* adding new features. Instead, please send your lovely pull requests and design ideas to more fully-featured libraries like [DeepMind's Haiku](https://github.com/deepmind/dm-haiku) and Google Research's [FLAX](https://github.com/google/flax) or [Trax](https://github.com/google/trax)."
```
Comment (danielsuo) [inline jax/experimental/README.md:10]: Quick fix: extra `"` at the end of the paragraph

**Comment by mattjj** [comment]
Thanks for getting this done, Paige!

**Comment by shoyer** [comment]
Looks great!

Maybe also a note specifically in a comment at the top of the source for Stax? That seems to be mostly relevant experimental sub-project falling under this category.

File: jax/experimental/README.md (hunk 1 above)
Comment (dynamicwebpaige) [reply jax/experimental/README.md:10]: @danielsuo: Am not sure why we would need a trailing `"` at the end of the paragraph - am I missing an opening quote? :)

Comment (gnecula) [inline jax/experimental/README.md]: This reminds me that we should mention these NN libraries in the main README. 

I would suggest dropping the "DeepMind" and "Google Research's" library attributions.

Comment (danielsuo) [reply jax/experimental/README.md]: Haha, I just meant there's an extra `"` that renders at the end of the paragraph i.e., tiny typo (image below). Great contribution from yours truly, I know!

image

Comment (dynamicwebpaige) [reply jax/experimental/README.md]: SGTM! Dropping. :)

Comment (dynamicwebpaige) [reply jax/experimental/README.md]: Ah, gotcha! Thanks, @danielsuo.

File: jax/experimental/README.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ hundred lines of code, so take a look inside and adapt them as you need!
 
 ### Neural-net building with Stax
 
+#### 👉 **Note**: These mini-libraries are meant to be an _inspiration_, but not a prescription.
```
Comment (hawkinsp) [inline jax/experimental/README.md:10]: Perhaps limit this text to apply just to Stax for now? It's not clear what the policy is for other libraries in `experimental`.

File: jax/experimental/README.md [hunk 3]
```diff
@@ -7,6 +7,10 @@ hundred lines of code, so take a look inside and adapt them as you need!
 
 ### Neural-net building with Stax
 
+#### 👉 **Note**: These mini-libraries are meant to be an _inspiration_, but not a prescription. 
+
+To serve that purpose, it is best to keep their code samples minimal; so we generally *will not merge PRs* adding new features. Instead, please send your lovely pull requests and design ideas to more fully-featured libraries like [Haiku](https://github.com/deepmind/dm-haiku), [FLAX](https://github.com/google/flax), or [Trax](https://github.com/google/trax).
```
Comment (mattjj) [inline jax/experimental/README.md:12]: Looks like the official capitalization is "Flax" not "FLAX" (even though on our end we use "JAX" not "Jax"). Tweak the capitalization?

File: jax/experimental/README.md (hunk 2 above)
Comment (dynamicwebpaige) [reply jax/experimental/README.md]: SGTM! Made a modification to refer to only Stax.

File: jax/experimental/README.md (hunk 3 above)
Comment (dynamicwebpaige) [reply jax/experimental/README.md]: Done! :)

**Comment by hawkinsp** [comment]
Thanks!

**Pull request merged**